cmd/frontend/graphqlbackend/org.go

```diff
@@ -283,6 +284,29 @@ func (r *schemaResolver) UpdateOrganization(ctx context.Context, args *struct {
 	return &OrgResolver{db: r.db, org: updatedOrg}, nil
 }
 
+// RemoveOrganization removes an org and all resources associated to it. Depending on the flag value, it will hard delete or soft delete the org.
+func (r *schemaResolver) RemoveOrganization(ctx context.Context, args *struct {
+	Organization graphql.ID
+}) (*EmptyResponse, error) {
+	flags := featureflag.FromContext(ctx)
+	orgHardDeletionEnabled := flags.GetBoolOr("org-hard-deletion", false)
+
+	orgID, err := UnmarshalOrgID(args.Organization)
+	if err != nil {
+		return nil, err
+	}
+
+	if !orgHardDeletionEnabled {
+		return nil, errors.New("hard deleting organization is not supported")
+	}
+
+	if err := database.Orgs(r.db).HardDelete(ctx, orgID); err != nil {
```

Comment:
You likely want this otherwise you'll hit a panic once the feature flags mocking is fixed.

```suggestion
	if err := r.db.Orgs().HardDelete(ctx, orgID); err != nil {
```

Reply:
thanks, @unknwon . That fixed the issue and saved from a rabbit hole. 🎉